packages/paste-website/src/pages/getting-started/engineering/index.mdx

```diff
@@ -92,17 +81,31 @@ The list of external third party dependencies are as follows:
   </Table>
 </Box>
 
-Install them using:
-
 ```shell
+# yarn
 yarn add react react-dom prop-types
+
+# npm
+npm install react react-dom prop-types
 ```
 
-#### Install Paste System packages
+<Box marginBottom="space100" />
+
+### Paste Packages
+
+<Callout variant="information">
+  <CalloutTitle>Our packages are published on NPM</CalloutTitle>
+  <CalloutText>
+    Check out <Anchor href="https://docs.npmjs.com/about-npm">the NPM docs</Anchor> to learn more about these commands
+    and how these packages are installed.
+  </CalloutText>
+</Callout>
 
-Paste also requires these system dependencies:
+Paste has two packages to get started: `@twilio-paste/core` and `@twilio-paste/icons`.
+Paste Core includes all the component and libraries, while Paste Icons
+includes all the icon illustrations. [Read about how Paste keeps your bundles small](#how-the-core-package-stays-lean).
```

Comment:
The "Read about..." link here feels abrupt here. Maybe intro it with something Paste's performance.

Reply:
Rewording this section